[FreeBSD][build failure] FreeBSD 12-CURRENT build failure

```
-- Building osqueryd: /wrkdirs/usr/ports/sysutils/osquery/work/.build/osquery/osqueryd
[314/316] /usr/bin/c++  -DBOOST_NETWORK_ENABLE_HTTPS -DFREEBSD -DFREEBSD_12_0 -DNDEBUG -DOSQUERY_BUILD_DISTRO=12.0 -DOSQUERY_BUILD_PLATFORM=freebsd
-DOSQUERY_BUILD_SDK_VERSION=2.6.0 -DOSQUERY_THRIFT="" -DOSQUERY_THRIFT_LIB=thrift -DOSQUERY_THRIFT_POINTER=boost -DOSQUERY_THRIFT_SERVER_LIB=thrift/server -DSTRIP_FLAG_HELP=1
-DTHRIFT_SQUELCH_CONSOLE_OUTPUT=1 -I/usr/local/include -isystem /usr/local/osquery/legacy/include -isystem /usr/local/osquery/include -I/usr/local/osquery/include/openssl
-I/wrkdirs/usr/ports/sysutils/osquery/work/osquery-2.6.0/third-party/sqlite3 -I/wrkdirs/usr/ports/sysutils/osquery/work/osquery-2.6.0/include
-I/wrkdirs/usr/ports/sysutils/osquery/work/osquery-2.6.0 -Igenerated/gen-cpp -O2 -pipe -fstack-protector -isystem /usr/local/include -fno-strict-aliasing  -isystem
/usr/local/include -O2 -pipe -fstack-protector -isystem /usr/local/include -fno-strict-aliasing  -isystem /usr/local/include    -std=c++14 -stdlib=libc++ -Qunused-arguments
-Wstrict-aliasing -Wno-mi
 ssing-field-initializers -Wno-unused-local-typedef -Wno-deprecated-register -Wno-unknown-warning-option -Wnon-virtual-dtor -Wchar-subscripts -Wpointer-arith -Woverloaded-virtua
l -Wformat -Wformat-security -Werror=format-security -Wabi-tag -fpermissive -fstack-protector-all -pipe -fdata-sections -ffunction-sections -Os -fPIE -fpie -fPIC -fpic    
-march=x86-64 -mno-avx -Wall -Wextra -pedantic -Wuseless-cast -Wno-c99-extensions -Wno-zero-length-array -Wno-unused-parameter -Wno-gnu-case-range -MD -MT
osquery/CMakeFiles/shell.dir/devtools/shell.cpp.o -MF osquery/CMakeFiles/shell.dir/devtools/shell.cpp.o.d -o osquery/CMakeFiles/shell.dir/devtools/shell.cpp.o -c
/wrkdirs/usr/ports/sysutils/osquery/work/osquery-2.6.0/osquery/devtools/shell.cpp
FAILED: osquery/CMakeFiles/shell.dir/devtools/shell.cpp.o
/usr/bin/c++  -DBOOST_NETWORK_ENABLE_HTTPS -DFREEBSD -DFREEBSD_12_0 -DNDEBUG -DOSQUERY_BUILD_DISTRO=12.0 -DOSQUERY_BUILD_PLATFORM=freebsd -DOSQUERY_BUILD_SDK_VERSION=2.6.0
-DOSQUERY_THRIFT="" -DOSQUERY_THRIFT_LIB=thrift -DOSQUERY_THRIFT_POINTER=boost -DOSQUERY_THRIFT_SERVER_LIB=thrift/server -DSTRIP_FLAG_HELP=1 -DTHRIFT_SQUELCH_CONSOLE_OUTPUT=1
-I/usr/local/include -isystem /usr/local/osquery/legacy/include -isystem /usr/local/osquery/include -I/usr/local/osquery/include/openssl
-I/wrkdirs/usr/ports/sysutils/osquery/work/osquery-2.6.0/third-party/sqlite3 -I/wrkdirs/usr/ports/sysutils/osquery/work/osquery-2.6.0/include
-I/wrkdirs/usr/ports/sysutils/osquery/work/osquery-2.6.0 -Igenerated/gen-cpp -O2 -pipe -fstack-protector -isystem /usr/local/include -fno-strict-aliasing  -isystem
/usr/local/include -O2 -pipe -fstack-protector -isystem /usr/local/include -fno-strict-aliasing  -isystem /usr/local/include    -std=c++14 -stdlib=libc++ -Qunused-arguments
-Wstrict-aliasing -Wno-missing-fiel
 d-initializers -Wno-unused-local-typedef -Wno-deprecated-register -Wno-unknown-warning-option -Wnon-virtual-dtor -Wchar-subscripts -Wpointer-arith -Woverloaded-virtual -Wformat
-Wformat-security -Werror=format-security -Wabi-tag -fpermissive -fstack-protector-all -pipe -fdata-sections -ffunction-sections -Os -fPIE -fpie -fPIC -fpic -march=x86-64
-mno-avx -Wall -Wextra -pedantic -Wuseless-cast -Wno-c99-extensions -Wno-zero-length-array -Wno-unused-parameter -Wno-gnu-case-range -MD -MT
osquery/CMakeFiles/shell.dir/devtools/shell.cpp.o -MF osquery/CMakeFiles/shell.dir/devtools/shell.cpp.o.d -o osquery/CMakeFiles/shell.dir/devtools/shell.cpp.o -c
/wrkdirs/usr/ports/sysutils/osquery/work/osquery-2.6.0/osquery/devtools/shell.cpp
/wrkdirs/usr/ports/sysutils/osquery/work/osquery-2.6.0/osquery/devtools/shell.cpp:1082:32: error: qualified reference to 'SQL' is a constructor name rather than a type in this
context
  auto results = osquery::SQL::SQL(
                               ^
/wrkdirs/usr/ports/sysutils/osquery/work/osquery-2.6.0/osquery/devtools/shell.cpp:1188:34: error: qualified reference to 'SQL' is a constructor name rather than a type in this
context
    auto results = osquery::SQL::SQL(
                                 ^
2 errors generated.
ninja: build stopped: subcommand failed.
*** Error code 1

Stop.
```

--- a/osquery/devtools/shell.cpp
+++ b/osquery/devtools/shell.cpp
@@ -1079,7 +1079,7 @@ inline void meta_schema(int nArg, char** azArg) {
 }
 
 inline void meta_features(struct callback_data* p) {
-  auto results = osquery::SQL::SQL(
+  auto results = osquery::SQL(
       "select * from osquery_flags where (name like 'disable_%' or name like "
       "'enable_%') and type = 'bool'");
   for (const auto& flag : results.rows()) {

